src/api/config/sumeragi-params.md

```diff
@@ -0,0 +1,107 @@
+# Sumeragi Parameters
+
+Explain sumeragi module
+
+## `sumeragi.trusted_peers`
+
+::: warning Discussion
+
+Hey, is sumeragi.trusted_peers actually required to be set in the config?
+
+I guess that's the only way to specify the relevant topology and let the nodes know what addresses to communicate with as well as what public keys to use to verify respective node signatures
```

Comment:
No guessing, either yes or no please.

Reply:
This text is not for the users... it is for us to discuss.